stubs/untangle/untangle.pyi

```diff
@@ -0,0 +1,37 @@
+from _typeshed import Incomplete
+from collections.abc import Iterator
+from typing import Any
+from xml.sax import handler
+
+def is_string(x: Any) -> bool: ...
+
+class Element:
+    children: Incomplete
+    is_root: bool
+    cdata: str
+    def __init__(self, name: str | None, attributes: dict[str, Any] | None) -> None: ...
+    def add_child(self, element: Element) -> None: ...
+    def add_cdata(self, cdata: str) -> None: ...
+    def get_attribute(self, key: str) -> Any: ...
+    def get_elements(self, name: Incomplete | None = ...) -> list[Element]: ...
+    def __getitem__(self, key: str) -> Any: ...
+    def __getattr__(self, key: str) -> Element: ...
+    def __hasattribute__(self, name: str) -> bool: ...
+    def __iter__(self) -> Iterator[Element]: ...
+    def __bool__(self) -> bool: ...
+    __nonzero__: Incomplete
+    def __eq__(self, val: object) -> bool: ...
+    def __dir__(self) -> list[str]: ...
+    def __len__(self) -> int: ...
+    def __contains__(self, key: str) -> bool: ...
+
+class Handler(handler.ContentHandler):
+    root: Incomplete
+    elements: Incomplete
+    def __init__(self) -> None: ...
+    def startElement(self, name: str, attributes: dict[str, Any]) -> None: ...
+    def endElement(self, name: str) -> None: ...
+    def characters(self, cdata: str) -> None: ...
+
+def parse(filename: str, **parser_features: dict[str, bool]) -> Element: ...
```

Comment:
`dict[str, bool]` here means that every keyword argument needs to be a dictionary with string keys and bool values. I think that's probably not what you meant ;)

```suggestion
def parse(filename: str, **parser_features: bool) -> Element: ...
```

Reply:
ah, you're right, that one always catches me out...